File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/AdminDto.java
package com.afpa.cda.dto;

import java.util.List;

import com.afpa.cda.entity.Role;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
@ToString
public class AdminDto {
	private int id;
	private String nom;
	private String prenom;
	private String email;
	private String login;
	private String password;
	private String adresse;
	private List <RoleDto> roles;
	
	
}

File: Sources/gestion-culturelle-front/project/src/app/model/adminDto.ts
import { RoleDto } from './roleDto';

export class AdminDto {
    id: number;
    nom: string;
    prenom: string;
    email: string;
    login: string;
    password: string;
    adresse: string;
    role: RoleDto;

    constructor(
        id?: number,
        nom?: string,
        prenom?: string,
        email?: string,
        login?: string,
        password?: string,
        adresse?: string,
        role?: RoleDto,
    ) {
        this.id = id;
        this.nom = nom;
        this.prenom = prenom;
        this.email = email;
        this.login = login;
        this.prenom = prenom;
        this.password = password;
        this.adresse = adresse;
        this.role = role;


    }
}


